extra/mender-client-docker-addons/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN (cd /src/mender-configure-module && \
 RUN mkdir --parents /mender-install/etc/mender
 
 WORKDIR /src/mender
-RUN test -f CMakeLists.txt && cmake -D MENDER_NO_BUILD=1 -S .
+RUN if [ -f CMakeLists.txt ]; then cmake -D MENDER_NO_BUILD=1 -S .; fi
```

Comment:
I thought failing was the intention here. Why do we want it to succeed if `CMakeLists.txt` is missing?

Reply:
> I thought failing was the intention here. 

Why the `test`, then?

> Why do we want it to succeed if `CMakeLists.txt` is missing?

Right now it fails because it builds from `master` branch from mender. You have a point in that `integration` and `mender` repos are both versioned together and we could _assume_ there is a `CMakeLists.txt`.

For now I'll keep the `if` to have green pipeline and we can clean it up later on after we get the rid of the feature branches. It is harmless as is.

Reply:
Ref `test`: Of course, I was thinking of this totally backwards. You're right.